agreementbot/agreementbot.go

```diff
@@ -384,6 +384,40 @@ func (w *AgreementBotWorker) Initialize() bool {
 	return true
 }
 
+func (w *AgreementBotWorker) preprocessExistingAgreeements() {
```

Comment:
Where is this called? And, I thought you were going to do the checks on the agbot side as needed, instead of updating every agreement.

Reply:
Yeah, I think I changed to that partway through writing it and didn't call the function from either spot. It is fixed now